Chamasma/DesignSystem/Sources/DesignSystem/Components/ComponentViews/OrderButton.swift

```diff
@@ -0,0 +1,42 @@
+//
+//  OrderButton.swift
+//
+//
+//  Created by Mariam Ormotsadze on 10.05.23.
+//
+
+import Common
+import SwiftUI
+
+struct OrderButton: View {
```

Comment:
Please use vertical spacing between components, for example:

HStack()

HStack()

Reply:
Try to create some view properties from the code like:

var exampleView: some View { ... }

Reply:
I changed that, please have a look at that when you will have time

Reply:
No it is not changed. It's the same